### DISCUSSION TITLE
Information on stats

### DISCUSSION THREAD

**Discussion by medellin9** [ericaltendorf/plotman#640] [General]
Hey,

In the plotman interactive window (shown in the picture) you can see stats on active plots and most statistics speak for themselves. But I dont know what the different STAT (slp, run, dsk), USER, SYS mean.

I assume IO means time lost on input/output?

![image](https://user-images.githubusercontent.com/84143360/119267017-586b4380-bbed-11eb-833c-1845b64840ba.png)

**Comment by XerrorYT** [comment]
Perfectly normal.
https://github.com/ericaltendorf/plotman/wiki/Plot-statuses

**Reply by leonidcan** [reply]
User is the amount of CPU time spent in user-mode code (outside the kernel) within the process. This is only actual CPU time used in executing the process. Other processes and time the process spends blocked do not count towards this figure.

Sys is the amount of CPU time spent in the kernel within the process. This means executing CPU time spent in system calls within the kernel, as opposed to library code, which is still running in user-space. Like 'user', this is only CPU time used by the process. See below for a brief description of kernel mode (also known as 'supervisor' mode) and the system call mechanism.

**Reply by medellin9** [reply]
Ok thank you I googled kernel mode and user mode and it makes more sense now but the user time is higher than walled time?

**Reply by leonidcan** [reply]
LOL that's pretty easy. It counts time per CPU thread as 100% if you have 4 threads it will show 400% CPU time max. It will go down as next phases will use only one thread but average will always be higher than wall time

**Reply by medellin9** [reply]
I'm still learning, but thank you for your reply, it gave me some clarification.
